docs/sources/visualizations/node-graph.md

```diff
@@ -91,9 +91,9 @@ Optional fields:
 | Field name    | Type          | Description                                                                                                                                                                                              |
 | ------------- | ------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | title         | string        | Name of the node visible in just under the node.                                                                                                                                                         |
-| subTitle      | string        | Additional, name, type or other identifier that will be shown right under the title.                                                                                                                     |
-| mainStat      | string/number | First stat shown inside the node itself. Can be either string in which case the value will be shown as it is or it can be a number in which case any unit associated with that field will be also shown. |
-| secondaryStat | string/number | Same as mainStat but shown right under it inside the node.                                                                                                                                               |
+| subtitle      | string        | Additional, name, type or other identifier that will be shown right under the title.                                                                                                                     |
+| mainstat      | string/number | First stat shown inside the node itself. Can be either string in which case the value will be shown as it is or it can be a number in which case any unit associated with that field will be also shown. |
```

Comment:
```suggestion
| mainstat      | string/number | First stat shown inside the node itself. It can either be a string showing the value as is or a number. If it is a number,  any unit associated with that field is also shown. |
```